### PR TITLE
Update version numbers for release 1.3.0

### PR DESCRIPTION
**Pull request by markcowl**

### CODE REVIEW

**Comment by markcowl** [comment]
on demand run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/639/

sign job here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-sign/330/

File: src/ResourceManager/Cdn/Commands.Cdn.Test/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -45,5 +45,5 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
-[assembly: AssemblyVersion("1.0.0")]
-[assembly: AssemblyFileVersion("1.0.0")]
+[assembly: AssemblyVersion("1.0.1")]
```
Comment (hovsepm) [inline src/ResourceManager/Cdn/Commands.Cdn.Test/Properties/AssemblyInfo.cs]: This is the first version of Azure Cdn. I think we should release it as 1.0

File: tools/AzureRM/AzureRM.psd1 [hunk 2]
```diff
@@ -46,36 +46,36 @@ ProcessorArchitecture = 'None'
...
-    @{ ModuleName = 'AzureRM.UsageAggregates'; MaximumVersion = '1.9999.9999.9999'; ModuleVersion = '1.0.5'},
-    @{ ModuleName = 'AzureRM.Websites'; MaximumVersion = '1.9999.9999.9999'; ModuleVersion = '1.0.5'}
+    @{ ModuleName = 'AzureRM.Profile'; MaximumVersion = '1.9999.9999.9999'; ModuleVersion = '1.0.6'},
+    @{ ModuleName = 'Azure.Storage'; MaximumVersion = '1.9999.9999.9999'; ModuleVersion = '1.0.6'},
+    @{ ModuleName = 'AzureRM.ApiManagement'; MaximumVersion = '1.9999.9999.9999'; ModuleVersion = '1.0.6'},
+    @{ ModuleName = 'AzureRM.Automation'; MaximumVersion = '1.9999.9999.9999'; ModuleVersion = '1.0.6'},
+    @{ ModuleName = 'AzureRM.Backup'; MaximumVersion = '1.9999.9999.9999'; ModuleVersion = '1.0.6'},
+    @{ ModuleName = 'AzureRM.Batch'; MaximumVersion = '1.9999.9999.9999'; ModuleVersion = '1.0.6'},
+    @{ ModuleName = 'AzureRM.Compute'; MaximumVersion = '1.9999.9999.9999'; ModuleVersion = '1.2.5'},
+    @{ ModuleName = 'AzureRM.Cdn'; MaximumVersion = '1.9999.9999.9999'; ModuleVersion = '1.0.1'},
```
Comment (hovsepm) [inline tools/AzureRM/AzureRM.psd1]: version 1.0?

File: src/ServiceManagement/Services/Commands.Utilities/Azure.psd1 [hunk 3]
```diff
@@ -9,7 +9,7 @@
 @{
 
 # Version number of this module.
-ModuleVersion = '1.2.3'
+ModuleVersion = '1.3.0'
```
Comment (hovsepm) [inline src/ServiceManagement/Services/Commands.Utilities/Azure.psd1]: Storage reference should be 1.0.6. Looks like ASM script has a bug.

**Comment by hovsepm** [comment]
@markcowl , Shouldn't we bum up minor version for Azure.Storage package?

**Pull request merged**